Documentation/gettingstarted/encryption.rst

```diff
@@ -200,6 +201,58 @@ used by Cilium. The key variable is a uint8 with value between 0-16 and should
 be monotonically increasing every re-key with a rollover from 16 to 0. The
 Cilium agent will default to KEYID of zero if its not specified in the secret.
 
+.. _encryption_ipam_eni:
+
+Encryption with IPAM ENI mode
+==============================
+
+In order to enable encryption on EKS with the IPAM ENI mode, deploy cilium via
+Helm with the following options:
+
+.. parsed-literal::
+
+    helm install cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set ipam.mode=eni \\
+      --set tunnel=disabled \\
+      --set encryption.enabled=true \\
+      --set encryption.nodeEncryption=false
+
+By default, this enables routing via the ``cilium_host`` interface.
+Specifying encryption interfaces with ``--set encryption.interface=ethX`` is not supported.
+
+Validate the Setup
+==================
+
+Run a ``bash`` shell in one of the Cilium pods with ``kubectl -n kube-system
+exec -ti <cilium pod> -- bash`` and execute the following commands:
+
+1. Install tcpdump
+
+.. code:: bash
```

Comment:
```suggestion
.. code-block:: shell-session
```
(Possibly with `$`s at the beginning of the commands)
(Same suggestion further down in the doc)